src/analytics.js

```diff
@@ -13,6 +13,34 @@ export function trackEvent(category, action, label, value) {
 	}
 }
 
+export function trackPageView(pageName = null) {
+	if (window.DEBUG) {
+		log('trackPageView', pageName);
+		return;
+	}
+	if (window.ga) {
+		if (pageName){
+			ga('send', 'pageview', pageName);
+		} else {
+			ga('send', 'pageview');
```

Comment:
传空参数和不传参数有区别吗？

Reply:
While technically the send command for pageview hits accepts an optional page field as the third parameter, passing the page field that way is not recommended when tracking single page applications. This is because fields passed via the send command are not set on the tracker—they apply to the current hit only. Not updating the tracker will cause problems if your application sends any non-pageview hits (e.g. events or social interactions), as those hits will be associated with whatever page value the tracker had when it was created.

from : https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications#handling_multiple_urls_for_the_same_resource


文档的建议是，先设置，然后在发无参数的pageView